daemon/api_apps_test.go

```diff
@@ -180,7 +180,8 @@ func (s *appsSuite) TestGetAppsInfo(c *check.C) {
 	svcNames := []string{"snap-a.svc1", "snap-a.svc2"}
 	for _, name := range svcNames {
 		s.SysctlBufs = append(s.SysctlBufs, []byte(fmt.Sprintf(`
-Id=snap.%s.service
+Id=snap.%[1]s.service
```

Comment:
nitpick: there's no need to change this line, as `%s` continues to work.

Reply:
Fixed.